winml/adapter/WinMLAdapterFactory.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#ifdef __cplusplus
+#include <WinMLAdapter.h>
+using namespace Windows::AI::MachineLearning::Adapter;
+
+#else
+struct IWinMLAdapter;
+typedef struct IWinMLAdapter IWinMLAdapter;
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+__declspec(dllexport) HRESULT STDMETHODCALLTYPE OrtGetWinMLAdapter(IWinMLAdapter** adapter);
```

Comment:
__declspec(dllexport)  is not needed.

Reply:
Good point!

Reply:
But is dllimport still needed?
For Windows 32bits build, do you need to specify calling convention(_stdcall/...) ?

Reply:
I'm going to add ORT_EXPORT